rsmtool/rsmexplain.py

```diff
@@ -36,6 +36,47 @@
 from .utils.wandb import init_wandb_run, log_configuration_to_wandb
 
 
+def verify_config_features(explain_config, rsmtool_config, feature, logger=None):
```

Comment:
Hmm, I am not a fan of passing the entire configuration objects to a  function and then modifying one of the passed objects in memory as well as returning it. In fact, now that I think about it, I think we probably don't need a function at all. Apologies for having you do all this work but it didn't really become clear to me until I saw it implemented.